products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Amazon RDS for MySQL
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+
+releases:
+-   releaseCycle: "8.0"
+    releaseDate: 2018-10-23
+    eol: 2026-04-01
+    latest: "8.0.32"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.7"
+    releaseDate: 2016-02-22
+    eol: 2023-10-01
+    latest: "5.7.41"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.6"
+    releaseDate: 2013-07-01
+    eol: 2022-03-01
+    latest: "unknown"
+
+---
+
+> [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql) is a PaaS offering from Amazon for
+> creating managed MySQL Community Edition databases. RDS makes it easier to set up, operate, and
+> scale MySQL deployments on AWS cloud.
+
+**MySQL recommends that all users run the latest available minor release for whatever major version
+is in use.**
+
+- AWS will provide support for major releases 3 years after their RDS release date.
+
+- AWS will provide support for minor versions 1 year after their RDS release date.
+
+Keep in mind that by default minor versions are automatically upgraded during maintenance windows.
+
+For more info on how RDS versions are deprecated see the AWS [documentation](https://aws.amazon.com/rds/faqs/#What_happens_when_an_Amazon_RDS_DB_engine_version_is_deprecated.3F).
```

Comment:
The content of this link should be included (and if possible resumed) in the description. The link must be kept though.

Reply:
I've written out most of the important details now from this link.